plugins/kafka/kafka.ecllib

```diff
@@ -212,9 +212,10 @@ EXPORT Consumer(VARSTRING topic, VARSTRING brokers = 'localhost', VARSTRING cons
         END;
 
         // Set the offset for each partition on each node
-        result := PROJECT
+        result := NORMALIZE
             (
-                distOffsets,
+                NOFOLD(distOffsets),
```

Comment:
I don't think the change from project to normalize is needed (and confuses the code).  The reason the distribute is removed is as follows:

    count(filter(local-count-project(distribute(dataset)))
is converted to
   count(filter(project(distribute(dataset))))
to
    count(distribute(filter(project(dataset)))
to
   count(filter(project(dataset)))

That is only a problem if it matters which node SetMessageOffsets is called from and eclcc doesn't know that information.

Having the nofold in place is probably the simplest fix - although the new NOCOMBINE will allow more optimizations.